### DISCUSSION TITLE
diary/cheng-xu-yuan-ti-gao-fang-shi/

### DISCUSSION THREAD

**Discussion by giscus** [alex306314/alex306314.github.io#3] [blog-comments]
# diary/cheng-xu-yuan-ti-gao-fang-shi/

产品上想多一点，设计上想多一点，技术上做深一点，思维框架上学多一点。

https://aqrun.com/diary/cheng-xu-yuan-ti-gao-fang-shi/

**Comment by aqrun** [comment]
comment data test
